app/posts/finishing-touches-for-forms.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Finishing touches for forms
+date: 2025-01-05
+description: Some tips on making sure forms written in Ruby on Rails are accessible
+author:
+  name: Peter Yates
+  url: https://github.com/peteryates
+---
+
+The [GOV.UK formbuilder](https://govuk-form-builder.netlify.app) makes it easy for Ruby on Rails developers to build forms that follow the rules set out in the [GOV.UK design system](https://design-system.service.gov.uk/).
+
+It takes care of the markup, provides a nice API for customising the fields to meet the needs of your service and takes care of clearly dispalying any error messages.
```

Comment:
Suggestion for avoiding repeated “takes care of”. Plus maybe HTML is clearer than “markup”?

```suggestion
It generates the correct HTML, provides a nice API for customising the fields to meet the needs of your service and takes care of clearly displaying any error messages.
```